src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/unit/restapi/controller/ReferralControllerTest.kt

```diff
@@ -445,7 +482,7 @@ constructor(
     }
 
     every { referralService.getReferralsByOrganisationId(organisationId, pageable, null, null, null) } returns
-      PageImpl(projectionsForFirstReferral.toApi() + projectionsForSecondReferral.toApi(), pageable, 2L)
+      PageImpl(projectionsForFirstReferral.toApi(prisoners, prisons, organisationId) + projectionsForSecondReferral.toApi(prisoners, prisons, organisationId), pageable, 2L)
```

Comment:
See how `.toApi()` is now more complicated than a simple transformation? Looks more like a builder or something. Possibly hard to parse for newcomers to the project.

Reply:
Spoken about and resolved.